content/advanced_usage/one_click_backup.md

```diff
@@ -63,7 +75,22 @@ The restoration process is not automated and needs to be done manually. Please r
 
     > **For example:** A backup using GoCD version 19.3 cannot be restored onto GoCD version 19.2. See the file `version.txt` in the backup directory to know the version of GoCD that was used to perform the backup
 
-- Database - Unzip the `db.zip` found in the backup directory. Unzip will create a file called `cruise.h2.db` . Copy this file to the directory `${SERVER_INSTALLATION_DIR}/db/h2db` .
+- Database - This depends on the kind of database being used:
+
+  - If you're using H2: Unzip `db.zip` and copy the file `cruise.mv.db` found inside it to the directory `${SERVER_INSTALLATION_DIR}/db/h2db`.
+
+  - If you're using PostgreSQL: Use `psql` to restore using the database backup file (for instance `db.gocd`). This depends on how you connect to the database. Example:
+
+        psql -d my_new_db -h my.db.host.com --password <db.gocd
+
```

Comment:
Shouldn't it be `pg_restore`. `psql` is when the `--format=plain` is used as backup args.
https://extensions-docs.gocd.org/postgresql/current/backup-restore-database/#step-2-restore-the-backup-file

Reply:
I brought up a Postgres DB and took a backup. It gave me a plain-text backup. This was with Postgres 12. What I showed there worked. Isn't the default plain?

Maybe: When the DB stuff changed, this changed too?

Reply:
Yes, you are right - the default is `plain`(https://www.postgresql.org/docs/12/app-pgdump.html). For our addon - it wasn't. 
Sorry got confused.